dev/com.ibm.ws.jaxrs.2.0_fat/publish/servers/com.ibm.ws.jaxrs.fat.bookcontinuationstore/server.xml

```diff
@@ -0,0 +1,16 @@
+<server>
+    <featureManager>
+        <feature>jaxrs-2.0</feature>
+    </featureManager>
+
+
+  	<include location="../fatTestPorts.xml"/>
+  	<javaPermission className="java.io.FilePermission" name="ALL FILES" actions="read"/>
+  	<javaPermission className="java.util.PropertyPermission" name="org.apache.cxf.Logger" actions="read"/>
+  	<javaPermission className="java.util.PropertyPermission" name="org.apache.cxf.JDKBugHacks.*" actions="read"/>
+    <javaPermission className="java.lang.RuntimePermission" name="accessClassInPackage.sun.misc"/>
+    <javaPermission className="java.lang.RuntimePermission" name="accessDeclaredMembers"/>
+    <javaPermission className="java.lang.RuntimePermission" name="getClassLoader"/>
+    <javaPermission className="java.lang.RuntimePermission" name="setContextClassLoader"/>
+    <javaPermission className="javax.security.auth.AuthPermission" name="getPolicy"/>
```

Comment:
These permissions seem excessive... are we including CXF in the app?  If not, can you open an issue for somebody to investigate these?  There shouldn't be any reason for a user to need all these permissions to use the CXF shipped in Liberty.

Reply:
We are packaging CXF in the app